layers/+window-management/perspectives/README.org

```diff
@@ -164,6 +164,4 @@ Set the variable =perspective-display-help= to =nil=
 
 If you changed stuff in your ~@Home~ perspectives, the state won't be reloaded 
 each time you restart Emacs, each time the Home buffer will be shown. If you don't want
- to see the home buffer at startup change the value of ~spacemacs-persp-show-home-at-startup~.
-
-
+ to see the home buffer at startup change the value of ~spacemacs-persp-show-home-at-startup~ to ~nil~.
```

Comment:
OK it makes more sense, but `nil` or `t` still does not show the home buffer, it is shown temporarily but then `.spacemacs` is opened automatically.

@CestDiego does this behavior sound familiar to you ?